app/pulp/app/models/progress.py

```diff
@@ -54,6 +56,7 @@ class ProgressReport(Model):
     done = models.IntegerField(default=0)
 
     task = models.ForeignKey("Task", on_delete=models.CASCADE)
+    suffix = models.TextField(default=None)
```

Comment:
i've read on django docs that it is not recommended to use null=true for string based fields https://docs.djangoproject.com/en/1.10/ref/models/fields/#null
maybe @seandst can provide more info here?